Palaso.MSBuildTasks/GenerateReleaseArtifacts.cs

```diff
@@ -132,7 +146,7 @@ private static void ConvertMarkdownLineToChangelogLine(string markdownLine, List
 		/// <returns></returns>
 		internal bool StampMarkdownFileWithVersion()
 		{
-			if(StampMarkdownFile)
+			if (StampMarkdownFile && Release)
```

Comment:
I don't think this is a necessary change, if the developer has requested that the MarkdownFile is updated why should we reject it if they didn't also set Release?